examples/eslint/BUILD.bazel

```diff
@@ -0,0 +1,58 @@
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@npm//eslint:index.bzl", "eslint_test")
+
+# Make file available to lint call below
+exports_files(
+    [
+        ".eslintrc.js",
+    ],
+)
+
+# The files listed here are auto generated by rules_nodejs
+# We don't want them linted
+write_file(
+    name = "eslintignore",
+    out = ".eslintignore",
+    content = [
+        "lint_loader.js",
+        "lint_require_patch.js",
+        "_lint.module_mappings.json",
+        "lint.sh",
+    ],
+)
+
+write_file(
+    name = "write_eslint_chdir_script",
+    out = "eslint_chdir.js",
+    content = [
+        "process.chdir(__dirname)",
+    ],
+)
+
+eslint_test(
+    name = "lint",
+    args = [
+        # Bazel invokes node binaries from the root of the monorepo
+        "--node_options=--require=./$(rootpath :write_eslint_chdir_script)",
+        "--max-warnings 0",
+        "--ext=js",
+        "--config=.eslintrc.js",
+        "src",
+    ],
+    data = [
+        # This will automatically be read by eslint
+        ":eslintignore",
+        ":write_eslint_chdir_script",
+        ".eslintrc.js",
+        # Needed for Bazel coverage
+        "@npm//c8",
+        # ESLint config and dependencies
+        "@npm//eslint-config-airbnb",
```

Comment:
I understand why `airbnb` is here but why the other dependencies? doesn't eslint already depend on them?

Reply:
they are peer dependencies of the airbnb config